src/sans_fitter/pr_inversion.py

```
@@ -0,0 +1,1349 @@
+    'invert',
+    'auto_invert',
+    'estimate_alpha',
+    'estimate_n_terms',
+    'explore_dmax',
+    'PrResult',
+    'DmaxScan',
+    'AlphaEstimate',
+    'NTermsEstimate',
+    'InsufficientDataError',
+    'PrEstimationError',
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)